src/db.ts

```diff
@@ -381,6 +381,10 @@ export class DB {
 			const req = new ProtoCreateBranchRequest().setProject(this.name).setBranch(name);
 			this.grpcClient.createBranch(req, (error, response) => {
 				if (error) {
+					if (error.code === Status.UNKNOWN) {
```

Comment:
`Status.NOT_FOUND`

Reply:
Please add a similar [unit test](https://github.com/tigrisdata/tigris-client-ts/blob/main/src/__tests__/tigris.rpc.spec.ts#L1022-L1031)

Reply:
In a sample project, I specified the name of a non-existing project in the .env file and console.log the error returned.
It returned UKNOWN.

I understand the mocked rpc version returns Status.NOT_FOUND hence my tests were working as expected.

But I'm positive the useful error message we specified will not show when the library is used against the actual server.

I'd suggest we check for both (error.code === Status.UNKNOWN or status.NOT_FOUND) but I'm leaving that judgement to you because I don't know if the server has been updated to return the right status when the project does not exist.

Reply:
You are right. For branching operations on a non-existing project, it does throw `Error: 2 UNKNOWN: not found`. And for operations in a collection context, it throws `Error: 5 NOT_FOUND: project doesn't exist 'catalog'`.

Good find. This needs to be a consistent error from Tigris, tigrisdata/tigris#1244